### PR TITLE
[Add Artwork] Saudi Deno

### PR DESCRIPTION
**Pull request by X7md**
image

Preview:
Saudi Deno

### CODE REVIEW

File: data/artwork.json [hunk 1]
```diff
@@ -309,5 +309,19 @@
       "github": "Arc-blroth"
     },
     "license": "CC-BY-SA-4.0"
+  },
+  {
+    "date": "2022-08-03",
+    "image": "/images/artwork/snow_den.png",
```
Comment (kt3k) [inline data/artwork.json:315]: This seems pointing to a wrong file?

Looks like you forgot to commit the actual image file?

Comment (X7md) [reply data/artwork.json]: My bad

Comment (X7md) [reply data/artwork.json]: Done, add image in `./images/artwork/`

**Review by kt3k — APPROVED** [review]
LGTM

**Pull request merged**